index.py

```diff
@@ -174,7 +222,7 @@ def calculate_doc_length(self):
         for _, posting_list in self.dictionary.items():
             for posting in posting_list.postings:
                 if posting.doc_id not in self.doc_lengths:
-                    posting_weight = 1 + math.log(len(posting.positions), 10)
+                    posting_weight = 1 + math.log(len(posting.positions), 10) # SHOULD BE IN FRONT??
```

Comment:
wait is there something off here lol?

Reply:
ohya, i thought that that line should come before the previous line "if posting.doc_id not in self.doc_lengths:" since posting_weight is needed for both if else condition

Reply:
hmm yea thats right.